src/main/java/com/amazon/opendistroforelasticsearch/knn/plugin/script/KNNScoringUtil.java

```diff
@@ -64,6 +104,35 @@ public static float cosinesimilOptimized(float[] queryVector, float[] inputVecto
         return (float) (dotProduct / (Math.sqrt(normalizedProduct)));
     }
 
+    /**
+     * Whitelisted method for users that can be used script to avoid repeated calculation of normalization
```

Comment:
Nit: "Whitelisted cosineSimilarity method that can be used in a script to avoid repeated calculation of normalization for the query vector.

Example:
...
"

Reply:
Ack